packages/api-server/src/plugins/api.ts

```diff
@@ -0,0 +1,63 @@
+import fastifyUrlData from '@fastify/url-data'
+import type { Options as FastGlobOptions } from 'fast-glob'
+import type { FastifyInstance } from 'fastify'
+import fastifyRawBody from 'fastify-raw-body'
+
+import type { GlobalContext } from '@redwoodjs/context'
+import { getAsyncStoreInstance } from '@redwoodjs/context/dist/store'
+// @ts-expect-error TODO(jtoar)
+import { coerceRootPath } from '@redwoodjs/fastify-web/helpers'
```

Comment:
Could use some help with this one; not sure why TS doesn't seem to recognize the subpath imports

Reply:
Simplifying the explanation a bit, but, basically this package needs to be ESM for that to work. 
... and if we make this ESM we also need to make Internal ESM